packages/redux/src/raffles/actions/factories/modifyRaffleParticipationFactory.ts

```diff
@@ -0,0 +1,54 @@
+import * as actionTypes from '../../actionTypes';
```

Comment:
We usually name PATCH actions by "update" instead of "modify"

Reply:
done, pls double check

Reply:
You probably forgot to push, everything is still called "modify" 🙈

Reply:
can you double check pls ?!